light-client/poseidon/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name         = "poseidon"
```

Comment:
you can move the whole `poseidon` crate under `light-client` and import inside `relayer` workspace.